aten/src/ATen/native/mkldnn/xpu/Blas.cpp

```diff
@@ -277,73 +280,6 @@ Tensor baddbmm(
   return r;
 }
 
-Tensor& addbmm_out(
```

Comment:
Does Intel GPU not support `addbmm_out`?

Reply:
We does not need to register `addbmm`, as cuda/cpu/xpu share same implementation by calling `addmm`. They share an entry at `natives_functions.yaml`